tests/unit/bases/test_almalinux.py

```diff
@@ -0,0 +1,1587 @@
+#
+# Copyright 2021-2023 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# pylint: disable=duplicate-code
+
+import subprocess
+from datetime import datetime
+from pathlib import Path
+from textwrap import dedent
+from unittest.mock import ANY, call, patch
+
+import pytest
+from logassert import Exact  # type: ignore
```

Comment:
Can you make this ignore comment more specific?

Reply:
That's not possible, `# type: ignore`  is the recommended way to ignore missing typed library.

https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker